### DISCUSSION TITLE
How to change the default homepage/landing page of site?

### DISCUSSION THREAD

**Discussion by jbfi1** [hugo-fixit/FixIt#452] [Q&A]
Hi, I would like to change the default homepage when you open my site to have /posts as homepage and remove the default homepage. Am I supposed to edit one of the *.html files in the theme config or change the baseURL? Im having trouble figuring it out.

This example is how I would like my site to look like when opened [https://ayang.ink/](https://ayang.ink/), source code: [https://github.com/AyangHuang/Ayang-Blog](https://github.com/AyangHuang/Ayang-Blog) As opposed to landing on the homepage like this by default, which I want to omit completely: [https://fixit.lruihao.cn](https://fixit.lruihao.cn), source code: [https://github.com/hugo-fixit/FixIt?tab=readme-ov-file.](https://github.com/hugo-fixit/FixIt?tab=readme-ov-file.)

**Comment by Lruihao** [comment]
This is a very strange requirement, but if you really want it, it's also very simple. You just need to create `posts/_index.md` and write the content:

```md
---
title: All Posts
url: /
---
```

**Comment by Shivang9983** [comment]
{{ define "main" }}
<div class="page home">
    <div class="archive">
        
        {{ range .Site.RegularPages }}
            {{ if eq .Section "posts" }}
                <article class="archive-item">
                    <a href="{{ .RelPermalink }}" class="archive-item-link">{{ .Title }}</a>
                    <span class="archive-item-date">{{ .Date.Format "2006-01-02" }}</span>
                </article>
            {{ end }}
        {{ end }}
    </div>
</div>
{{ end }}

**Comment by sumitkumar2374** [comment]
```baseURL``` only defines the site's root URL, so it won't change which page is shown as the homepage.

If you want `/posts` to become the default landing page, you'll likely need to customize the theme or configure Hugo to redirect the home route to the posts list. It would also help to check whether the FixIt theme provides a setting to disable the default landing page.

Hopefully, someone familiar with the FixIt theme can confirm the recommended approach.
